qiling/extensions/r2/r2.py

```diff
@@ -121,6 +121,11 @@ def _cmd(self, cmd: str) -> str:
 
     def _cmdj(self, cmd: str) -> List[Dict]:
         return json.loads(self._cmd(cmd))
+    
+    def read(self, addr: int, size: int) -> bytes:
+        self._cmd(f"s {addr}")
+        hexstr = self._cmd(f"p8 {size}")
```

Comment:
I would have consolidated the two r2 commands into one: `self._cmd(f"p8 {size} @ {addr}")`